.github/workflows/nightly.yml

```diff
@@ -59,24 +79,52 @@ jobs:
           java-version: '11'
           distribution: 'adopt'
 
-      - name: Gradle cache
-        uses: actions/cache@v2
+      - name: Lint
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: lint
+
+      - name: Bundle the lint report
+        if: always()
+        run: find . -name lint-results\* | zip -@ -r lint-report.zip
+
+      - name: Upload the JVM lint report
+        if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: lint-report
+          path: lint-report.zip
+
+  android_tests:
+    runs-on: ubuntu-latest
+    name: Android CI checks
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
         with:
-          path: |
-            ~/.gradle/caches
-            ~/.gradle/wrapper
-          key: gradle-${{ runner.os }}-${{ hashFiles('**/*.gradle*') }}-${{ hashFiles('**/gradle/wrapper/gradle-wrapper.properties') }}-${{ hashFiles('**/buildSrc/**/*.kt') }}
+          submodules: recursive
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
 
       - name: Decode secret
         env:
           FLANK: ${{ secrets.FLANK }}
         run: echo $FLANK > flank.json
 
       - name: Build
-        run: ./gradlew androidTestsBuild
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: androidTestsBuild
 
       - name: Run Android Tests
-        run: ./gradlew runFlankAndroidTests
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: runFlankAndroidTests
 
       - name: Bundle the Android CI checks report
         if: always()
```

Comment:
should we update the reports to `android-tests-report` in this task and the next one?

Reply:
Nice catch! Is not important but definitely more consistent :)